internal/commands/extension_package.go

```diff
@@ -0,0 +1,38 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+// ExtensionPackageFlags define flags provided to the ExtensionPackage command
+type ExtensionPackageFlags struct {
+	PackageTomlPath   string
+	Format            string
+	Publish           bool
+	Policy            string
+	ExtensionRegistry string
+	Path              string
+}
+
+// Packager and PackageConfigReader to be added here and argument also to be added in the function
+
+// ExtensionPackage packages (a) extension(s) into OCI format, based on a package config
+func ExtensionPackage(logger logging.Logger, cfg config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "package <name> --config <config-path>",
+		Short: "Package an extension in OCI format.",
```

Comment:
```suggestion
		Short: "Package an extension in OCI format",
```
(nit, to make it more consistent with other commands... `pack buildpack` also has this inconsistency, but we could at least make it better here)